### PR TITLE
Adding Mini project resources-issue#115

### LINKED ISSUE #115: Can I add some mini project Ideas and resources to practice for beginners(HTML,CSS,JS).

### PR DESCRIPTION
**Pull request by Uma-95**
fixes #115 
@DhanushNehru could you review my PR. Looking forward to your feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -328,6 +328,16 @@ More information about the pack can be found here [Github Student Developer Pack
 - [Resources 5](https://join.codecademy.com/learn/introduction-to-javascript/)
 - [Resources 6](https://www.freecodecamp.org/news/learn-javascript-free-js-courses-for-beginners/)
 
+## Mini Projects using HTML,CSS and Javascript
```
Comment (epicadidash) [inline README.md:331]: Change title to `tutorials of creating mini projects using html , css , js ` @Uma-95

**Review by epicadidash — CHANGES_REQUESTED** [review]
Check comments in the file

**Comment by DhanushNehru** [comment]
Please resolve conflicts @Uma-95

File: README.md (hunk 1 above)
Comment (Uma-95) [reply README.md]: @epicadidash Done.

**Comment by Uma-95** [comment]
@epicadidash kindly review the changes.

**Pull request closed without merging**